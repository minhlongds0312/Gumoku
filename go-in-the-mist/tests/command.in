place A1
place B1
place A2
place B2
place A3
place B3
place A4
place J12
place B4
place B12
place A423
invalidinput
view
history
who
term