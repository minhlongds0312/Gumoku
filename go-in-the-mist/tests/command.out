Invalid coordinate
Invalid!
A17,xxxxxxxxxx....xxx....xxx....xxx....xxx....xxx....
A1B1A2B2A3B3A4B4B12
W